redash/handlers/queries.py

```diff
@@ -271,3 +273,19 @@ def post(self, query_id):
         parameter_values = collect_parameters_from_request(request.args)
 
         return run_query(query.data_source, parameter_values, query.query_text, query.id)
+
+
+class QueryVersionListResource(BaseResource):
+    @require_permission('view_query')
+    def get(self, query_id):
+        results = models.Change.list_versions(models.Query.get_by_id(query_id))
+        page = request.args.get('page', 1, type=int)
+        page_size = request.args.get('page_size', 25, type=int)
+
+        return paginate(results, page, page_size, lambda q: q.to_dict(full=False))
```

Comment:
I don't see paginate options in the UI/html so I'm not sure how this paginate will react at query version 26?